README.md

```diff
@@ -172,12 +172,18 @@ If the config-helper doesn't provide the level of customization you need for you
 | EXISTING_VPC_NAME | If MLSpace is being deployed into an existing VPC this should be the name of that VPC (must also set `EXISTING_VPC_ID`) | - |
 | EXISTING_VPC_ID | If MLSpace is being deployed into an existing VPC this should be the id of that VPC (must also set `EXISTING_VPC_NAME`) | - |
 | EXISTING_VPC_DEFAULT_SECURITY_GROUP | If MLSpace is being deployed into an existing VPC this should be the default security group of that VPC | - |
+<<<<<<< HEAD
```

Comment:
Probably left over form merge

Reply:
Ahh thanks, I thought I completed the merge but I guess this snuck through